### PR TITLE
add admin page and superuser

### PR DESCRIPTION
**Pull request by RyukungG**
admin page แan create and edit questions. In question can fill in question text, damage, currency and add choice.

### CODE REVIEW

File: qa_rpg/admin.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
-# Register your models here.
+class ChoiceInline(admin.TabularInline):
+    model = Choice
+    extra = 3
+
+
+class QuestionAdmin(admin.ModelAdmin):
+    fieldsets = [
+        (None, {'fields': ['question_text']}),
+        ('Date information', {'fields': ['damage', 'currency'],
```
Comment (Halcyon905) [inline qa_rpg/admin.py]: date info??

Comment (RyukungG) [reply qa_rpg/admin.py]: fixed

**Review by Halcyon905 — APPROVED** [review]
seems good to me

File: qa_rpg/admin.py [hunk 2]
```diff
@@ -1,3 +1,22 @@
 from django.contrib import admin
+from .models import Question, Choice, Player
 
-# Register your models here.
+class ChoiceInline(admin.TabularInline):
+    model = Choice
+    extra = 3
```
Comment (MarojT) [inline qa_rpg/admin.py]: Should it be 2 choices at least for question?

**Review by MarojT — APPROVED** [review]

**Pull request merged**